.github/labeler.yml

```diff
@@ -0,0 +1,20 @@
+infrastructure:
+- .github/**/*
+- helper/**/*
+
+amd sev:
+- internal/shim-sev/**/*
```

Comment:
```suggestion
- internal/shim-kvm/**/*
```

Reply:
Shouldn't this also include the backend?

Also change at this path should probably label both as SEV and KVM